### PR TITLE
UI refresh

### PR DESCRIPTION
**Pull request by echaidemenos**
### Description

Updates the UI

TODO:

- [x]  move the "go to" and legend buttons to navbar
- [ ] disable the goto button while the boundaries are loading
- [x]  make the left panel have the "hover" appearance
- [x] update colors
- [x] Update map style
- [x] reduce height of the navbar
- [x] update layer menu colors (#1144 )

## Checklist - did you ...

Test your changes with

- [x] `REACT_APP_COUNTRY=rbd yarn start`
- [x] `REACT_APP_COUNTRY=cambodia yarn start`
- [x] `REACT_APP_COUNTRY=mozambique yarn start`
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

## Screenshot/video of feature:

Screenshot from 2024-03-19 14-35-43

### CODE REVIEW

File: frontend/src/context/tableStateSlice.ts [hunk 1]
```diff
@@ -89,12 +85,12 @@ export const isLoading = (state: RootState): boolean =>
...
 export const getCurrentData = (state: RootState): TableData =>
-  state.tableState.data || { columns: [], rows: [] };
+  state.tableState.data;
 
 export const tableErrorSelector = (state: RootState): string | undefined =>
   state.tableState.error;
 
 // export actions
-export const { hideTable } = tableStateSlice.actions;
+// export const {  } = tableStateSlice.actions;
```
Comment (ericboucher) [inline frontend/src/context/tableStateSlice.ts:94]: 👣 ?

File: frontend/src/muiTheme.tsx [hunk 2]
```diff
@@ -220,7 +225,20 @@ const theme: any = createTheme({
       root: {
         '&.Mui-disabled': {
           pointerEvents: 'auto',
+          color: '#282b2d',
```
[the remark this answers is not in the document]
Comment (echaidemenos) [reply frontend/src/muiTheme.tsx:228]: i was experimenting with the disabled goto button color. now it seems this appears nowhere, so i am removing it.

**Review by ericboucher — APPROVED** [review]

**Comment by ericboucher** [comment]
@echaidemenos lgtm, let's just fix this issue:

Screenshot 2024-03-12 at 2 26 12 PM

**Comment by wadhwamatic** [comment]
@echaidemenos @ericboucher - Overall this is looking great. I found one small issue. If I run an analysis and then return to the layers tab, the analysis result layer in the list still shows the older UI style:
Screenshot 2024-04-17 at 17 48 09

I'm also still not so convinced about the look for all caps in the UI (layers, charts, analysis). How much effort is it to try without all caps here?

**Review by wadhwamatic — APPROVED** [review]
Thanks for the fixes and updates. Good to go from my side. @ericboucher - we can hold until we hear from Marc about the font size and capitalization, or go ahead and revert later if this needs to change. I'll leave that to you. =)

**Comment by wadhwamatic** [comment]
@ericboucher - Marc is onboard for the font size and case changes we made. This is GTG!

**Pull request merged**